pext/qml/AboutDialog.qml

```diff
@@ -40,7 +40,8 @@ Dialog {
                                "nl": [{"name": "Sylvia van Os", "email": "sylvia@hackerchick.me"},
                                       {"name": "Heimen Stoffels", "email": "vistausss@outlook.com"}],
                                "ru": [{"name": "Nelly Simkova", "email": "stylinsonnelly@gmail.com"}],
-                               "zh_TW": [{"name": "Jeff Huang", "email": "s8321414@gmail.com"}]}
+                               "zh_TW": [{"name": "Jeff Huang", "email": "s8321414@gmail.com"}]},
+                               "ar": [{"name": "Ramy Gamal", "email": "ramyeg26@gmail.com"}]}
```

Comment:
Could you put it at the start to keep the alphabetical order of the list?

Reply:
Yes Sure.